projects/m3/001-total-the-values/js/index.js

```diff
@@ -0,0 +1,18 @@
+
+
+// Requiring prompt-sync to use prompt in terminal mode.
+const prompt = require('prompt-sync')();
+
+
+function totalInput() {
+
+    let value = prompt("Please, enter a sequence of values to see their sum (enter a blank line to stop): ");
```

Comment:
Why did you use **let** vs **const**?
https://www.freecodecamp.org/news/var-let-and-const-whats-the-difference/

Reply:
Used the wrong variable declaration, tried to fixed that along the milestone.